### PR TITLE
Pack generic modules from Synthea into module-builder

### LINKED ISSUE #118: Load modules from Synthea
Modules are currently loaded from mocks. They should be loaded from the Synthea repository instead.

### PR DESCRIPTION
**Pull request by arscan**
@kjmahalingam @jawalonoski Added the build script for modules, put in some error checking and instructions in the README.  Let me know if it is self explanatory and works for you.  I didn't update the modules from Synthea yet, but it should be ready for Jay to give it a shot.

Fixes #118

### CODE REVIEW

**Comment by jawalonoski** [comment]
@dehall

**Comment by kjmahalingam** [comment]
This build script worked and I was able to follow the directions in the README.

This unfortunately wouldn't update the deployed version of the module builder whenever the modules in Synthea are changed. That is a of course a more difficult task, so it seems that a manual update will be necessary every so often.

File: README.md [hunk 1]
```diff
@@ -9,6 +9,18 @@ Currently supports:
...
+## Synchronizing Generic Modules in Synthea
+
+This application currently embeds all generic modules within the application.  In order to 
+update these modules to match the most recent ones available in the Synthea repository,
+run the `build-modules` script, pointing to the modules directory within a local copy of `Synthea`.
+
+```sh
+  npm run build-modules ../synthea/src/main/resources/modules # Point to the modules directory of synthea
+  git status # This should have changed ./src/data/modules.js
+  npm test # Run tests to ensure the new modules file valid
```
Comment (jawalonoski) [inline README.md]: So after this, just...
```
git add -R .
git commit -m "Module Update."
git push
```

???

Comment (arscan) [reply README.md]: Yes.  You need to commit the changes.

Comment (arscan) [reply README.md]: Probably a PR first ;-)

**Pull request merged**